Challenge1.exs

```diff
@@ -0,0 +1,40 @@
+defmodule Challenge do
+  def num_divisible_by_3_sequence(a, b) do
+    if a >= 1 and a <= b and b < 231 do
+      sequence = ["1"]
+      ini = 1
+      con = 0
+      sequence = construct_sequence(sequence, ini, a, b, con)
+      IO.inspect(sequence)
+      IO.puts("Number of divisibles by 3 elements in the sequence: #{Enum.reduce(Enum.filter(sequence, fn element -> rem(String.to_integer(element), 3) == 0 end), 0, fn _element_, acc -> acc + 1 end)}")
+    else
+      IO.puts("Valid Numbers: (a >= 1 y b < 231)")
```

Comment:
👍🏽

Reply:
You can move 1, or 231 to external module attributes (or as input params) for avoiding using magic numbers ⭐

Reply:
Same for sequence initial values, ini and con

Reply:
![code](https://user-images.githubusercontent.com/87148694/196577749-6c257950-a13b-4176-81e3-ea3e117528cd.png)